types/arc4/index.d.ts

```diff
@@ -0,0 +1,115 @@
+/// <reference types="node" />
+
+export type ArcAlgorithm = "arc4" | "rc4+" | "rc4a" | "vmpc";
+
+type AlgoToClass<T extends ArcAlgorithm> = T extends "arc4" ? Arc4
+    : T extends "rc4+" ? Rc4p
+    : T extends "rc4a" ? Rc4a
+    : T extends "vmpc" ? Vmpc
+    : never;
+
+export default function arc4<T extends ArcAlgorithm>(
```

Comment:
Not sure how this came into declaration. Module seems to be plain, classic CJS:
https://unpkg.com/browse/arc4@3.4.0/index.js
so more like (pseudocode) imo:
```ts
declare function arc4(): ...

declare namespace arc4 {
    function normal(): ...
    function lodash(): ...
}

export = arc4;
```
(include types in namespace to get them exported)

Reply:
But how do I fix the compiler error now?